projects/api/experiments/experiments.py

```diff
@@ -152,3 +157,38 @@ async def handle_delete_experiment(project_id: str,
     experiment = experiment_controller.delete_experiment(experiment_id=experiment_id,
                                                          project_id=project_id)
     return experiment
+
+
+@router.get("/{experiment_id}/{task_name}/logs/eventsource")
+async def handle_log_deployment(experiment_id: str,
+                                req: Request,
+                                task_name: str):
+    """
+    Handles log event source requests to /<experiment_id>/<task_name>/logs/eventsource.
+
+    Parameters
+    ----------
+    experiment_id : str
+    req : fastapi.Request
+    task_name: str
+
+    Returns
+    -------
+    EventSourceResponse
+    """
+    task_name = urllib.parse.unquote_plus(task_name)
+    run_id = get_latest_run_id(experiment_id)
+    pods = list_workflow_pods(run_id)
+    if len(pods) > 0:
```

Comment:
Aqui poderia ser um booleano, algo do tipo if pods. 
_"_1. if not seq:
In Python, empty lists evaluate False, and non-empty lists evaluate True in boolean contexts. Therefore, you can simply treat the list as a predicate returning a Boolean value. This solution is highly Pythonic and recommended by the PEP8 style guide._"_
https://www.techiedelight.com/check-list-empty-python/